ios/ElectronCash/electroncash_gui/ios_native/history.py

```diff
@@ -347,9 +347,7 @@ def _synch_full(self, wallet):
                 tx.deserialize()
                 ins = tx.inputs()
                 for x in ins:
-                    xa = x['address']
-                    if isinstance(xa, PublicKey):
-                        xa = xa.toAddress()
+                    xa = x.get('address')
```

Comment:
Lol.. I guess I didn't know what I was doing and just copy-pasta'd here to get it working. *sheepish look*

Reply:
Actually everything *should* have 'address' property set right now (at very least to None) because of how parse_sig works. But in future I would like to have that assumption dropped so this is just a bit of forward thinking.